sipuadart/lib/sdp/attributes/attribute.dart

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
Shouldn't this file be named `Attribute.dart` as opposed to `attribute.dart`?

Reply:
I don't remember the Dart convention regarding file names. But I notice some are starting with lower case and others are beggining with upper case. I'm guessing we are not going according here to the convention due to this lack in consistency?

Reply:
It's Dart convertion naming files in lower-snakecase for files (aaa_bbb.dart) and its class names as capitalized snakecase.

Reply:
What got me confused is that we have some files starting with lowercase and others starting with uppercase.

E.g. `attribute.dart` vs `Category.dart`

Shouldn't all files always start with lowercase or are there exceptions to this rule?

Reply:
Ooooooohhh, now I see. My mistake. I was convinced that I had written all files in snakecase. Tks